src/app/ApplicationTemplate.UWP/App.xaml

```diff
@@ -20,7 +20,6 @@
 				<ResourceDictionary Source="Views/Styles/Controls/DataValidationView.xaml" />
 				<ResourceDictionary Source="Views/Styles/Controls/ExtendedSplashScreen.xaml" />
 				<ResourceDictionary Source="Views/Styles/Controls/FlipViewItem.xaml" />
-				<ResourceDictionary Source="Views/Styles/Controls/Frame.xaml" />
```

Comment:
We don't need to set the frame style anymore with the flag as default in the app.xaml